woodchipper/logger.py

```diff
@@ -31,7 +37,12 @@ def unbind(self, *keys: str) -> "BoundLogger":
     def try_unbind(self, *keys: str) -> "BoundLogger":
         return super().try_unbind(*self.prefix_keys(*keys))  # type: ignore
 
+    def clear_prefix(self):
+        self.key_prefix = ""
+
     def _proxy_to_logger(
         self, method_name: str, event: Optional[str] = None, *event_args: str, **event_kw: Any
     ) -> Any:
-        return super()._proxy_to_logger(method_name, event, *event_args, **self.prefix_kwargs(**event_kw))
+        return super()._proxy_to_logger(
+            method_name, event, *event_args, **(self.prefix_kwargs(**event_kw) if self.key_prefix else event_kw)
```

Comment:
The ternary here isn't needed. We do the same check in the prefix_kwargs method itself.

Reply:
Thanks! Fixed this in the most recent commit